MetaMask/metamask-extension
# Add support for localised Snap manifests

## **Description**

This adds support for localised Snap manifests. Instead of getting the Snap name from the subject metadata controller, I've added a new selector which returns the localised manifest.

Related to MetaMask/snaps#1508.

## **Manual testing steps**

1. Install `npm:@metamask/localization-example-snap` from test-snaps.
2. Make sure the proper name and description is shown everywhere.

## **Screenshots/Recordings**

<img width="472" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/7503723/03ce5e0a-a43a-4f5d-be91-ef75b15323ba">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
